pkg/types/types.go

```diff
@@ -14,16 +16,27 @@ type Header struct {
 
 // Block contains information of block.
 type Block struct {
-	Number      *big.Int `json:"number"`
-	Hash        string   `json:"hash"`
-	Timestamp   uint64   `json:"timestamp"`
-	ParentHash  string   `json:"parentHash"`
-	ReorgedHash string   `json:"reorgedHash"`
-	Logs        []Log    `json:"logs"`
+	Number      *big.Int  `json:"number"`
+	Hash        string    `json:"hash"`
+	Timestamp   uint64    `json:"timestamp"`
+	ParentHash  string    `json:"parentHash"`
+	ReorgedHash string    `json:"reorgedHash"`
+	Logs        []Log     `json:"logs"`
+	FullBlock   *pb.Block `json:"fullBlock"`
 }
 
 // Message ...
 type Message struct {
```

Comment:
In my opinion, we should define a protobuf type for `Message` to publish to `pubsub`, so it will be more consistent.

Reply:
make sense